### PR TITLE
README Suggestions

### PR DESCRIPTION
**Pull request by JesseEmond**
Minor modification suggestions to the `README`!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
-|-- CTF name
+|-- CTF_name
 |     |-- README.md (event description)
-|     +-- pwn 42
-|          |-- README.md (your solution)
-|          +-- Extra files
+|     +-- pwn_42
+|     |    |-- README.md (your solution)
+|     |    +-- Extra files
+      +-- simple_challenge.md (if it does not require other files)
```
Comment (DrPandemic) [inline README.md]: It misses a `|` at the beginning

File: README.md [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
 +-- Another CTF
       |--
```
Comment (DrPandemic) [inline README.md]: I know it was mine, but could you remove this line plz? It's ugly

File: README.md (hunk 1 above)
Comment (JesseEmond) [inline README.md]: It's not meant to be a folder though. Not sure what the formatting convention is being used here. Could you please show concretely what you would prefer?

File: README.md (hunk 2 above)
Comment (JesseEmond) [inline README.md]: Done

File: README.md (hunk 1 above)
Comment (JesseEmond) [inline README.md]: I see what you meant. Changing it.

**Pull request merged**